spec/controllers/api/v1/monsters_controller_spec.rb

```diff
@@ -43,10 +44,10 @@
           habitat: "City"
         }
       }
+      sign_in user
```

Comment:
Is there a reason for a user to be signed in for the monster index? Just a consideration to make to save memory from the test creating a user object if it's not being used

Reply:
We have restricted creation of monsters to only authenticated users.  When we run the test without signing in, the creation of the monster does not occur.